packages/local_auth/local_auth_darwin/darwin/Tests/FLALocalAuthPluginTests.m

```diff
@@ -38,6 +38,49 @@ - (LAContext *)createAuthContext {
 
 @end
 
+@interface StubAlertFactory : NSObject <FLADAlertFactory>
+#if TARGET_OS_OSX
+@property(copy, nonatomic) NSAlert *alert;
+- (instancetype)initWithNSAlert:(NSAlert *)alert;
+#else
+@property(copy, nonatomic) UIAlertController *alertController;
+- (instancetype)initWithUIAlertController:(UIAlertController *)alertController;
+#endif
+
+@end
+
+@implementation StubAlertFactory
+
+#if TARGET_OS_OSX
+
+- (instancetype)initWithNSAlert:(NSAlert *)alert {
+  self = [super init];
+  if (self) {
+    _alert = alert;
```

Comment:
These aren't doing copies, which violates the current API contract. (This will be resolved if you don't declare them as copy though.)